File: HealthCareApi/Models/HealthRecord.cs
using System;

namespace HealthCareApi.Models
{
    public class HealthRecord
    {
        public int RecordId { get; set; }
        public int AppointmentId { get; set; }
        public int PatientId { get; set; }
        public int DoctorId { get; set; }
        public DateTime VisitDate { get; set; }
        public string Diagnosis { get; set; }
        public string Prescription { get; set; }
        public string Notes { get; set; }
        public DateTime CreatedDate { get; set; }

        public virtual Appointment Appointment { get; set; }
        public virtual Patient Patient { get; set; }
        public virtual Doctor Doctor { get; set; }
    }
}

File: HealthCareAppPatientDoctor.Angular/src/app/core/models/health-record.ts
export interface HealthRecord {

  healthRecordId: number;

  patientId: number;

  patientName: string;

  doctorId?: number;

  doctorName?: string;

  appointmentId: number;

  visitDate: string;

  diagnosis: string;

  prescription: string;

  notes?: string;

}